com.github.calo001.luna.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.github.calo001.luna",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.calo001.luna",
+    "finish-args" : [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "Luna",
+            "buildsystem" : "meson",
+            "config-opts" : [
```

Comment:
Not needed. it's already the default value of libdir.

Reply:
Fixed :+1: